install/kubernetes/cilium/values.yaml

```diff
@@ -510,6 +510,36 @@ daemon:
   # -- Configure where Cilium runtime state should be stored.
   runPath: "/var/run/cilium"
 
+  # -- Configure a custom list of possible configuration override sources
+  # The default is "config-map:cilium-config,cilium-node-config,node"
```

Comment:
Do you think there are likely to be any security impacts of these defaults? It feels a bit like we'll have to make some assumptions about the ServiceAccounts/ClusterRoles that k8s application developers typically have and whether this could potentially allow unintentional reconfiguration of cilium-agent configurations on the node. For instance, if node annotations can be freely customized, then that same privilege could be used to disable network policies in Cilium.

Reply:
I had the same thought around node annotations a few days ago. The Gateway API folks have a good design for a security-sensitive priority-inverting overrides system, which is cool but I balked on implementing.

For now, I'll just disable node annotations by default.